CMakeLists.txt

```diff
@@ -74,5 +74,14 @@ IF(Trilinos_ENABLE_DataTransferKitMoabAdapters)
 ENDIF()
 
 ##---------------------------------------------------------------------------##
+## Check that there are no trailing whitespace
+##---------------------------------------------------------------------------##
+
+ADD_TEST(
+    NAME check_no_trailing_whitespace
+    COMMAND sh -c "grep --exclude=*.git* -rl '[[:blank:]]$' ${CMAKE_SOURCE_DIR} test $? -eq 1"
+)
```

Comment:
Oh you are right, we need to restrict to DTK source.  We can adjust and exclude any file pattern we like.